DEPS

```diff
@@ -41,6 +41,12 @@ vars = {
   # If there are problems with the toolchain, contact fuchsia-toolchain@.
   'clang_version': 'git_revision:20d06c833d833ef6b2d0f519cc4a7998d49a2803',
 
+  # The goma version and the clang version can be tightly coupled. If goma
+  # stops working on a clang roll, this may need to be updated using the value
+  # from the 'integration' tag of
+  # https://chrome-infra-packages.appspot.com/p/fuchsia/third_party/goma/client
+  'goma_version': ' git_revision:41b3bcb64014144a844153fd5588c36411fffb56',
```

Comment:
not `integration`?

Are we updating the clang roller to update this?

Reply:
Yeah that's a good idea. We can ask the Skia autoroller folks if that's possible.

Reply:
Not `integration` because we should probably avoid unpinned dependencies.